app/src/main/java/de/grobox/transportr/utils/IntentUtils.kt

```diff
@@ -86,22 +96,33 @@ object IntentUtils {
         val geo = Uri.parse(uri1 + uri2)
 
         // show station on external map
-        val intent = Intent(Intent.ACTION_VIEW)
-        intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
-        intent.data = geo
-        if (intent.resolveActivity(context.packageManager) != null) {
-            Log.d(context.javaClass.simpleName, "Starting geo intent: " + geo.toString())
-            context.startActivity(intent)
+        val intent = Intent(Intent.ACTION_VIEW).apply {
+            flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
+            data = geo
+        }
+        val intents = context.packageManager.queryIntentActivities(intent, 0).mapNotNull {
```

Comment:
bad news: https://cketti.de/2020/09/03/avoid-intent-resolveactivity/

but can stay like this in this PR as it was like this before.

Reply:
Yeah, we might stumble upon this again when targeting Android 11.